<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>JSONWidget</class>
 <widget class="QWidget" name="JSONWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>JSON Viewer</string>
  </property>
  <property name="styleSheet">
   <string notr="true">   /* 分割器样式 */
   QSplitter::handle {
       background-color: #ddd5db;
   }
   </string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QSplitter" name="mainSplitter">
     <property name="lineWidth">
      <number>1</number>
     </property>
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <property name="handleWidth">
      <number>6</number>
     </property>
     <property name="childrenCollapsible">
      <bool>false</bool>
     </property>
     <widget class="QWidget" name="leftWidget">
      <layout class="QVBoxLayout" name="leftVerticalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="Ui::CodeEditor" name="jsonTextEdit">
         <property name="frameShape">
          <enum>QFrame::Shape::NoFrame</enum>
         </property>
         <property name="lineWrapMode">
          <enum>QPlainTextEdit::LineWrapMode::NoWrap</enum>
         </property>
         <property name="placeholderText">
          <string>将JSON数据粘贴到这里!</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="middleWidget">
      <layout class="QVBoxLayout" name="middleVBoxLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QTreeView" name="jsonTreeView">
         <property name="frameShape">
          <enum>QFrame::Shape::NoFrame</enum>
         </property>
         <property name="lineWidth">
          <number>0</number>
         </property>
         <property name="editTriggers">
          <set>QAbstractItemView::EditTrigger::NoEditTriggers</set>
         </property>
         <property name="headerHidden">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="rightWidget">
      <layout class="QVBoxLayout" name="rightVBoxLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QTableWidget" name="propertiesTableWidget">
         <property name="frameShape">
          <enum>QFrame::Shape::NoFrame</enum>
         </property>
         <column>
          <property name="text">
           <string>Key</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Value</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Ui::CodeEditor</class>
   <extends>QPlainTextEdit</extends>
   <header>codeeditor.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
